pyTSEB/net_radiation.py

```diff
@@ -115,17 +113,17 @@ def calc_difuse_ratio(S_dn, sza, press=1013.25, SOLAR_CONSTANT=1320):
     ratiox[fclear > 0.88] = 0.88
     dirnir = (Rdirnir / potnir) * \
         (1. - ((.88 - ratiox) / .68)**.6667)  # Eq. 12
-    dirvis = np.maximum(0.0, dirvis)
-    dirnir = np.maximum(0.0, dirnir)
-    dirvis = np.minimum(1, dirvis)
-    dirnir = np.minimum(1, dirnir)
+
+    dirvis = np.clip(dirvis, 0.0, 1.0)
+    dirnir = np.clip(dirnir, 0.0, 1.0)
     difvis = 1.0 - dirvis
     difnir = 1.0 - dirnir
+
     return np.asarray(difvis), np.asarray(
         difnir), np.asarray(fvis), np.asarray(fnir)
 
 
-def calc_emiss_atm(ea, T_A_K):
+def calc_emiss_atm(ea, t_a_k):
```

Comment:
I know that by convention all capital variables refer to constants, but in this case it refers to a symbol from model equation. So for consistency with equations in published papers and in other places in the code (e.g. T_R_K - radiometric temperature in Kelvin, z_T - temperature measurement height) I would prefer if we leave it as T_A_K. There are plenty of other variables with similar capital names (e.g. H, LE, R_A) which I would also suggest to leave as they are.

Reply:
It's a bit confusing, given that all capital letters means top-level constant and most syntax highlighters will also color the variables as such, but if you believe it will hurt code readability more if we change it, we should keep it as all capital letters.